.circleci/config.yml

```
@@ -2,6 +2,30 @@ version: 2.1
 
 supported-eslint-versions: &supported-eslint-versions ["local", "7"]
 
+deploy_filters: &deploy_filters
+  filters:
+    branches:
+      ignore: /.*/
+    tags:
+      # Trigger on tags that begin with `v`
+      only: /^v.*/
+
+commands:
+  save_v1_cache:
+    description: Save cache for future build
+    steps:
+      - save_cache:
+          key: v1-dependencies-{{ checksum "yarn.lock" }}
+          paths:
+            - node_modules
```

```suggestion
            - ~/.cache/yarn
```

IME it's sometimes a little dangerous to cache `node_modules` (e.g. if for whatever reason the system architecture changes between CI builds, breaking native modules), but caching the global yarn cache is pretty harmless. Plus it's [what Circle CI recommends](https://circleci.com/docs/2.0/yarn/#caching).